### PR TITLE
Projects Page Finer Details

### PR DESCRIPTION
**Pull request by lankaukk**
Fixing a bunch of finer details on the projects page, so it doesn't look like it’s waiting for things to be added while we're still developing additional functionality. 

- adjusting the width of cards
- justifying cards so the last ones align with the grid
- project image is a grey gradient
- matching font sizes and colors to figma sketches 
- fix placement of sorting dropdown
- sorting dropdown trigger has grey background when menu is open

| Before | After|
| ------------- | ------------- |
| Screenshot 2024-03-04 at 7 10 47 PM | Screenshot 2024-03-04 at 7 07 36 PM  |
| Screenshot 2024-03-04 at 7 09 23 PM  | Screenshot 2024-03-04 at 7 08 40 PM  |

### CODE REVIEW

File: utopia-remix/app/components/sortProjectsContextMenu.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { useProjectsStore } from '../store'
 import { contextMenuDropdown, contextMenuItem } from '../styles/contextMenu.css'
 import { sprinkles } from '../styles/sprinkles.css'
 import { CheckIcon } from '@radix-ui/react-icons'
+import { start } from 'repl'
```
Comment (ruggi) [inline utopia-remix/app/components/sortProjectsContextMenu.tsx:7]: 🫥

File: utopia-remix/app/routes/projects.tsx [hunk 2]
```diff
@@ -373,6 +374,8 @@ const ProjectsHeader = React.memo(({ projects }: { projects: ProjectWithoutConte
     }
   }
 
+  const [sortMenuOpen, setSortMenuOpen] = useState(false)
```
Comment (ruggi) [inline utopia-remix/app/routes/projects.tsx:377]: it would be nice to stick to `React.useState` here (and drop the `useState` import)

**Review by ruggi — COMMENTED** [review]
\0/

**Review by ruggi — APPROVED** [review]

**Review by liady — APPROVED** [review]

**Review by balazsbajorics — APPROVED** [review]

File: utopia-remix/app/routes/projects.tsx [hunk 3]
```diff
@@ -852,11 +837,16 @@ const ProjectRow = React.memo(
 ProjectRow.displayName = 'ProjectRow'
 
 const ProjectCardActions = React.memo(({ project }: { project: ProjectWithoutContent }) => {
+  const [sortMenuOpen, setSortMenuOpen] = React.useState(false)
+
   return (
     <div style={{ display: 'flex', alignItems: 'center', gap: 5 }}>
-      <DropdownMenuRoot>
+      <DropdownMenuRoot onOpenChange={() => setSortMenuOpen(!sortMenuOpen)}>
```
Comment (seanparsons) [inline utopia-remix/app/routes/projects.tsx:844]: This should probably be wrapped in a `React.useCallback` function to prevent unnecessary re-renders.

**Review by seanparsons — APPROVED** [review]

**Pull request merged**